scripts/errcheck.sh

```diff
@@ -1,23 +1,23 @@
 #!/usr/bin/env bash
 
-# Check gofmt
-echo "==> Checking for unchecked errors..."
-
 if ! which errcheck > /dev/null; then
     echo "==> Installing errcheck..."
-    go get -u github.com/kisielk/errcheck
+    go install github.com/kisielk/errcheck@latest
 fi
 
-err_files=$(errcheck -ignoretests \
-                     -ignore 'github.com/hashicorp/terraform-plugin-sdk/helper/schema:Set' \
```

Comment:
We stop ignoring unchecked errors that come from the Schema set func. This is the main reason why we have so many issues :C